aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -836,11 +844,12 @@ module aptos_framework::fungible_asset {
 
     /// Used to delete a store.  Requires the store to be completely empty prior to removing it
     public fun remove_store(delete_ref: &DeleteRef) acquires FungibleStore, FungibleAssetEvents, ConcurrentFungibleBalance {
-        let store = &object::object_from_delete_ref<FungibleStore>(delete_ref);
-        let addr = object::object_address(store);
-        let FungibleStore { metadata: _, balance, frozen: _ }
+        let store = object::object_from_delete_ref<FungibleStore>(delete_ref);
+        let addr = object::object_address(&store);
+        let FungibleStore { metadata, balance, frozen}
             = move_from<FungibleStore>(addr);
         assert!(balance == 0, error::permission_denied(EBALANCE_IS_NOT_ZERO));
+        assert!(!frozen, error::permission_denied(ESTORE_IS_FROZEN));
```

Comment:
This assert looks new and not part of the event change that PR states. Any more context on this?

Reply:
maybe I should put it in another PR. I will delete this line